pygotham/frontend/about.py

```diff
@@ -1,28 +1,35 @@
 """About PyGotham."""
 
-from flask import Blueprint
+from collections import defaultdict
 
-from pygotham.frontend import direct_to_template
+from flask import Blueprint, render_template, url_for
+
+from pygotham.events import get_current
+from pygotham.models import AboutPage
 
 __all__ = ('blueprint',)
 
 blueprint = Blueprint('about', __name__, url_prefix='/about')
 
-direct_to_template(
-    blueprint,
-    '/code-of-conduct',
-    template='about/code-of-conduct.html',
-    endpoint='code_of_conduct',
-)
-direct_to_template(
-    blueprint,
-    '/privacy-policy',
-    template='about/privacy-policy.html',
-    endpoint='privacy_policy',
-)
-direct_to_template(
-    blueprint,
-    '/faq',
-    template='about/faq.html',
-    endpoint='faq',
-)
+
+def get_nav_links():
+    """Generates all about page titles and urls for use in the navbar."""
+    links = defaultdict(dict)
```

Comment:
For this and the previous OrderedDict comment: the rationale here was that link order is presentational, so sorting is done in the context processor. We may want to manually order certain links, though (a possible reason for a module defining this method), so perhaps the sorting done by the context processor should be restricted to automatically collected endpoints only and assume that anything returned from the `get_nav_links` functions is intentionally ordered?